.storybook/DocsDecorator/HealthFactor/fixture.ts

```diff
@@ -179,7 +179,7 @@ export const withAlpha = {
     done: true,
     reference: {
       url: 'https://paper.dropbox.com/doc/A11Y-Swarm-Promo--BL4ankwRPUBmg1CeqpcvamZ8Ag-Qi9uuHQyLm18NMCnUAtcO',
```

Comment:
Is this url just an example?

Reply:
oh, yes that is an example used in fixture data for testing purposes.
The HealthFactor object is here at .storybook/DocsDecorator/HealthFactors/metadata.json